subsys/usb/class/Kconfig

```diff
@@ -101,24 +101,25 @@ config USB_DEVICE_BLUETOOTH
 	help
 	  USB Bluetooth device class driver
 
+if USB_DEVICE_BLUETOOTH
+
 config USB_DEVICE_BLUETOOTH_VS_H4
 	bool "Enable USB Bluetooth H4 vendor command"
-	depends on USB_DEVICE_BLUETOOTH
-	select BT_HCI_RAW_H4
+	select USB_DEVICE_BT_H4
 	select BT_HCI_RAW_CMD_EXT
 	help
 	  Enables vendor command to switch to H:4 transport using the bulk
 	  endpoint.
 
 config USB_DEVICE_BT_H4
 	bool "USB Bluetooth H4 Device Class Driver"
-	select BT
```

Comment:
USB_DEVICE_BT_H4 does not depend on USB_DEVICE_BLUETOOTH. `subsys/usb/class/bt_h4.c` is independent class. Actually I think all the changes here are wrong.